pom.xml

```diff
@@ -63,7 +63,7 @@
         <skip.surefire.tests>${skipTests}</skip.surefire.tests>
         <skip.failsafe.tests>${skipTests}</skip.failsafe.tests>
         <!-- Set TLS-Attacker version once -->
-        <toolkit.tls.attacker.version>7.1.1</toolkit.tls.attacker.version>
+        <toolkit.tls.attacker.version>7.2.2-SNAPSHOT</toolkit.tls.attacker.version>
```

Comment:
we should always have a full tls-attacker-version in the dev repo for dependencies